spacy/__init__.py

```diff
@@ -46,12 +47,22 @@ def load(
     return util.load_model(name, disable=disable, exclude=exclude, config=config)
 
 
-def blank(name: str, **overrides) -> Language:
+def blank(
+    name: str,
+    *,
+    vocab: Union[Vocab, bool] = True,
```

Comment:
I do think `True` is the right default value to use here, for consistency with other places where we have used `None` or `False` to mean "not provided and also don't create it".

But what should we do if someone passes `vocab=None` or `vocab=False`? I guess we need to raise? Allowing `vocab=None` is one of those tiny steps towards the 'do what I mean' approach that gets us into trouble later.

Reply:
Sure, I'm not a fan of `True` but it is currently the default everywhere and how we do it in all other places.